src/configurations/index.js

```diff
@@ -2,6 +2,8 @@ import {userConfig} from "@/configurations/user";
 import {messagingConfig} from "@/configurations/messages";
 import {caseConfig} from "@/configurations/case";
 import {notificationConfig} from "@/configurations/notification";
+import {debateConfig} from "@/configurations/debate";
+import {rebuttalConfig} from "@/configurations/rebuttal";
```

Comment:
`rebuttalConfig` to be removed entirely after adjusting the changes.

Reply:
done